com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/message/Messages.java

```diff
@@ -76,7 +76,7 @@
     public static final String CANT_RUN_TASK_ON_APP = "Could not run task \"{0}\" on application \"{1}\"";
     public static final String CANT_CANCEL_TASK = "Could not cancel task with GUID \"{0}\"";
     public static final String CANT_UPDATE_SERVICE_PLAN = "Could not update service plan of service \"{0}\": \"{1}\"";
-    public static final String ERROR_SIZE_OF_UNCOMPRESSED_FILE_EXCEEDS_MAX_SIZE_LIMIT = "The size \"{0}\" of file \"{1}\" exceeds max size limit \"{2}\"";
+    
```

Comment:
Delete this empty line.

Reply:
The empty line exists for separation reasons, as the above messages are error messages, and the ones below are debug messages. Readability increases.